easybuild/easyconfigs/a/ARAGORN/ARAGORN-1.2.38-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'CmdCp'
+
+name = 'ARAGORN'
+version = '1.2.38'
+
+# HTTPS not working
+homepage = 'http://mbio-serv2.mbioekol.lu.se/ARAGORN/'
+description = "a program to detect tRNA genes and tmRNA genes in nucleotide sequences"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```

Comment:
@Darkless012 I think `iccifort` would be better here, if that works, since this may be beneficial in terms of performance?